qiskit/circuit/library/generalized_gates/isometry.py

```diff
@@ -341,15 +334,8 @@ def inv_gate(self):
 # Find special unitary matrix that maps [c0,c1] to [r,0] or [0,r] if basis_state=0 or
 # basis_state=1 respectively
 def _reverse_qubit_state(state, basis_state, epsilon):
-    state = np.array(state)
-    r = np.linalg.norm(state)
-    if r < epsilon:
-        return np.eye(2, 2)
-    if basis_state == 0:
-        m = np.array([[np.conj(state[0]), np.conj(state[1])], [-state[1], state[0]]]) / r
-    else:
-        m = np.array([[-state[1], state[0]], [np.conj(state[0]), np.conj(state[1])]]) / r
-    return m
+    res = isometry_rs.reverse_qubit_state(state, basis_state, epsilon)
+    return res
```

Comment:
Oh, that's a good point, I wasn't exactly sure how much of these were going to end up in rust when I was developing this. Having the python side was good for debugging test failures because I could inject prints that went to the python stdout buffer, but yeah let's drop the extra call overhead , I'll remove all these.

Reply:
Removed in: https://github.com/Qiskit/qiskit/pull/12197/commits/6e86efe470ba617e014aabd4c350c543ef5ecaf5